grails-app/migrations/0.8.x/changelog-2022-10-11-1200-add-default-product-type.xml

```diff
@@ -5,7 +5,7 @@
 
   <property name="now" value="now()"/>
 
-  <changeSet author="awalkowiak" id="0209202212000-0">
+  <changeSet author="awalkowiak" id="1110202212000-0">
```

Comment:
Why you made this change (and file name change)? If it is because you were getting an error that the changeset hash has changed, then you can add a `validCheckSum` property.

Reply:
yes, that was the error.
I didn't know that you can fix it by adding `validCheckSum` property.
Will fix it